app/src/main/java/io/homeassistant/companion/android/settings/SettingsFragment.kt

```diff
@@ -129,6 +129,10 @@ class SettingsFragment : PreferenceFragmentCompat(), SettingsView {
             return@setOnPreferenceClickListener true
         }
 
+        findPreference<Preference>("docs")?.let {
+            it.isCopyingEnabled = true
+        }
+
```

Comment:
If you set this in the `preferneces.xml` you don't need to set it here.

Reply:
Thanks for that!